src/resultsdisassemblypage.h

```diff
@@ -100,6 +106,10 @@ public slots:
     bool m_noShowAddress;
     // Switch Assembly to Intel Syntax
     bool m_intelSyntaxDisassembly;
+    // Default font size
+    int m_origFontSize;
+    // Version of objdump
+    QString m_objdumpVersion;
```

Comment:
unused in this patch, either remove or move to the patch where this is being used